src/styles/index.ts

```diff
@@ -4630,8 +4630,7 @@ const styles = (theme: ThemeColors) =>
         },
 
         workspaceTitleStyle: {
-            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE_BOLD,
-            fontWeight: '500',
+            ...headlineFont,
             fontSize: variables.workspaceProfileName,
```

Comment:
What size is this exactly? It should be the same default h1 size we use everywhere (which is 22px)

Reply:
rather than defining it as this though, why can't we just reuse our h1 styles? This way if we ever change our h1 styles, this will change too.

Reply:
I don't thin this conversation is resolved. Can you please respond to my comment above?

Reply:
Oh ok @shawnborton we have this items
    fontSizeh1: 19,
    fontSizeXLarge: 22,
which one should be used? a little bit confusing in case that h1 should be 22 and we are using 19.
Should it be changed? or we good to use fontSizeXLarge

Reply:
Oh interesting, from my understanding, our h1's everywhere should be using 22 as the font size. I guess just go with `fontSizeXLarge` for now...

Reply:
pushed changes to use directly `fontSizeXLarge `